app/projects/template.hbs

```diff
@@ -12,7 +12,7 @@
         options=customers
         placeholder="Select customer..."
         searchField="name"
-        onchange=(action (queue (action (mut selectedCustomer)) (action (mut selectedProject) null) (perform filterProjects)))
+        onchange=(action (queue (action (mut selectedCustomer)) (action (mut selectedProject) null) (action (mut selectedTask) null) (perform filterProjects)))
```

Comment:
Please make an action in the controller for this. It's obviously not readable..

Reply:
Done